.github/workflows/frontend.yml

```diff
@@ -90,6 +105,29 @@ jobs:
           aws-region: ${{ vars.AWS_REGION }}
           trunk-api-token: ${{ secrets.TRUNK_API_TOKEN }}
 
+  fe-tests-embedding-sdk-snippets:
+    needs: [files-changed]
+    if: |
+      !inputs.skip &&
+      needs.files-changed.outputs.frontend_embedding_sdk_sources == 'true'
+    runs-on: ubuntu-22.04
+    timeout-minutes: 20
+    steps:
+      - uses: actions/checkout@v4
+      - name: Prepare front-end environment
+        uses: ./.github/actions/prepare-frontend
+      - name: Prepare back-end environment
+        uses: ./.github/actions/prepare-backend
+        with:
+          m2-cache-key: "cljs"
+      - name: Compile CLJS
+        run: yarn build-pure:cljs
+        shell: bash
+      - name: Build Embedding SDK package
+        run: yarn build-embedding-sdk
+      - name: Run frontend embedding SDK snippets tests
+        run: yarn run test-unit:embedding-sdk-snippets
```

Comment:
I wonder why we decided to put it in `frontend.yml` instead of a SDK-specific workflow? I would prefer putting it in our dedicated workflow file for SDK-related tests

Reply:
Moved to `.github/workflows/embedding-sdk.yml`